docker-build.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf ./docker-deps && mkdir -p ./docker-deps
+cp -r /Users/yevheniishcherbina/go/src/github.com/Kava-Labs/ethermint docker-deps/
```

Comment:
Recommend making this a variable vs hard-coded, for example

https://stackoverflow.com/questions/40902445/using-variable-interpolation-in-string-in-docker

Reply:
It will be removed, it's temporary